implementations/app-typescript/src/__tests__/cases/000-sanity/output/types.ts

```diff
@@ -183,42 +184,102 @@ export interface TestImport_Module_Args_returnsArrayOfEnums {
 }
 
 /* URI: "testimport.uri.eth" */
-export const TestImport_Module = {
-  importedMethod: async (
+export abstract class BaseTestImport {
```

Comment:
This type isn't being used anywhere, is this what the user imports into their code? If so I don't think that's what we want, instead we'd want `import { TestImport } from ...`